src/Symfony/Bundle/FrameworkBundle/Secret/MutableSecretStorageInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Secret;
+
+/**
+ * MutableSecretStorageInterface defines an interface to add and update a secrets in a stores.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+interface MutableSecretStorageInterface extends SecretStorageInterface
```

Comment:
I think we should add a method and command to remove a secret as well. In theory people could also just delete the file but that is an implementation detail and the filename could be different (hashed) from the secret name.

Reply:
added a `secrets:remove` command